### DISCUSSION TITLE
ArchR multiome pipeline integrate Seurat object related information

### DISCUSSION THREAD

**Discussion by Jiahuan90** [GreenleafLab/ArchR#1455] [Questions / Documentation] [answered]
Hi rcores, 

I have ran multiome scRNA-seq and scATAC pipeline following the webpage successfully. However, I have a question: 

I use Seurat to run scRNA-seq part and already annotate cell types in Seurat object. But in the ArchR multiome pipeline, I have to use scRNA-seq filter Matrix as input, but not a Seurat object. Thus how could I integrate seurat related information to ArchR project? 

Thank you for your help~

**Comment by rcorces** [comment, marked as answer]
You would have to import this data into `cellColData` by matching cell names. Cells that dont match across the two platforms would probably need to be removed using `subsetArchRProject()`.
